ios/ElectrodeApiImpl/APIImpls/Navigation/ENNavigationUtils.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ENNavigationUtils.swift
+//  ElectrodeApiImpl
+//
+//  Created by Jeffrey Wang on 4/15/21.
+//  Copyright © 2021 Walmart. All rights reserved.
+//
+
+import Foundation
+
+open class ENNavigationUtils {
+
+    open func emitPayloadToCurrentRNView(navigationController: UINavigationController, jsonPayload: String?) {
+        if let miniAppNavVC = navigationController.viewControllers.last as? MiniAppNavViewController {
```

Comment:
We also, need to consider scenarios where a NavVC starting a custom VC instead of a MiniAppVC for a navigate call. This custom VC could be a native one. Lets discuss.

Reply:
Ok, not clear what you mean here, let's discuss